public/scss/pages/PlaceorderV2.module.scss

```
@@ -0,0 +1,476 @@
+
+  &:hover,
+  &:focus,
+  &:focus-visible,
+  &:active
+  {
+    outline: none;
+    border: solid 1px $color_black;
+    box-shadow: none;
+  }
+}
+
+.mapCenterButtonContainer
+{
+  position: absolute;
+  top: calc(100% + 12px);
+  right: 20px;
+  z-index: 2;
+}
+
+.mapCenterIcon
+{
+  color: $color_text_secondary;
+  height: 20px;
+  width: 20px;
+}
+
+.mapPopupCenterButton
+{
+  @include transition;
+  @include flex(row, center, center);
+  @include typographyBuilder(400, 16, 24);
+  position: relative;
+  margin: 0;
+  padding: 0 10px;
+  border: none;
+  box-shadow: none;
+  background-color: $color_white;
+  color: $color_black;
+  min-height: 40px;
+  box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.2);
+
+  &:hover,
+  &:focus-visible
+  {
+    background-color: $color_grey;
+  }
+}
+
+.mapFormAddress
+{
+  position: relative;
+  padding: 16px 20px 0;
+}
+
+.mapFooterContainer
+{
+  margin-top: 72px;
+
+  @media screen and (max-width: 479px)
+  {
+    margin-top: 0;
+    padding-top: 72px;
+  }
+
+  -moz-box-flex: 1;
+  flex-grow: 1;
+  overflow: auto;
+}
+
+.mapButtonFooterContainer
+{
+  width: 100%;
+  position: relative;
+}
+
+.mapButtonFooter
+{
+  @extend .btn_primary;
+  width: 100%;
+  height: 52px;
+  background-color: $color_black !important;
+  
+  &:hover
+  {
+    background-color: $color_black !important;
+  }
+}
+
+.mapDistrictLabel
+{
+  @include typographyBuilder(400, 16, 24);
+  padding: 0 20px;
+  margin: 17px 0 0;
+}
+
+.mapLabelAddress
+{
+  @include flex(row, center, flex-start);
+  @include fixedWidth(100%);
+  @include typographyBuilder(400, 12, 16);
+
+  position: relative;
+  margin: 0;
+  padding: 4px 20px;
+  text-transform: none;
+  color: $color_text_secondary;
+}
+
+.mapChangeAddressButton
+{
+  @include typographyBuilder(400, 14, 24);
+  border: none;
+  background: transparent;
+  padding: 0;
+  color: $color_blue;
+}
+
+.mapContainer
+{
+  @include flex(row-reverse, center, space-between);
+  border: solid 1px $color_elements_placeholder;
+  border-radius: 2px;
+
+  [class*="mapArea"]
+  {
+    @include fixedSize(133px);
+
+    @media screen and (max-width: 479px)
+    {
+      @include fixedSize(100px);
+    }
+
+    @media (max-width: #{$breakpoint_max_md})
+    {
+      @include fixedSize(133px);
+    }
+  }
+}
+
+.mapArea
+{
+  & > div
+  {
+    border: none !important;
+    padding: 15px;
+
+    & > div
+    {
+      @include fixedSize(103px !important);
```

```suggestion
      @include fixedSize(103px, true);
```

kalo tydac salah biar jadi `!important` tuh ada di parameter kedua pacc, dibikin jadi `true`